### PR TITLE
Fix error when using sent_ball on non-singleplayer server

### PR DESCRIPTION
**Pull request by callumok2004**
Fixes `Trying to remove server entity sent_ball#335 on client!`

### CODE REVIEW

File: garrysmod/lua/entities/sent_ball.lua [hunk 1]
```diff
@@ -141,9 +141,9 @@ end
 
 function ENT:Use( activator, caller )
 
-	self:Remove()
+	if ( activator:IsPlayer() ) and ( ( game.SinglePlayer() and CLIENT ) or SERVER ) then
```
Comment (Kefta) [inline garrysmod/lua/entities/sent_ball.lua:144]: Just change this check to SERVER, entities should never be removed on the client.

**Review by Kefta — CHANGES_REQUESTED** [review]

**Comment by Kefta** [comment]
Add the parentheses back around the if-condition and this is good to go.

**Comment by robotboy655** [comment]
I cannot reproduce such an issue.

https://wiki.facepunch.com/gmod/ENTITY:Use
`ENTITY:Use` is not even called on the client by the base game as far as I am aware.

**Comment by robotboy655** [comment]
Closing, as this tries to fix a non-existent issue as far as I can tell, and I see no replies to my comment above.

**Pull request closed without merging**